DPNUnet.imjoy.html

```diff
@@ -455,7 +450,7 @@
         options = await api.showDialog( name="Prediction Configurations", ui= "<br>".join([
         #"Image file name { id: 'image_file_name', type:'string', placeholder: 'nuclei.png'}",
         #"Mask file name { id: 'mask_file_name', type:'string', placeholder: 'nuclei_border_mask.png'}",
-         "Average nuclei size { id: 'nuclei_size', type:'number', placeholder: 20}",
```

Comment:
Here we should not simply set to 200, it will only work for our provided sample.

I think the default should be no-scale (20), if you want to make it compatible with our dataset, you should do a simple test check if it's actually our example dataset at least, the set the default value to 200.

Reply:
changed this in the next commit. inverted the order, first data-set is specified then default size of nucleus. also updated the docs for the prediction.